tools/eval.py

```diff
@@ -17,96 +18,118 @@
 from mindocr.models import build_model
 from mindocr.postprocess import build_postprocess
 from mindocr.metrics import build_metric
-from mindocr.utils.callbacks import Evaluator
+from mindocr.utils.evaluator import Evaluator
+from mindocr.utils.logger import get_logger
+
 
 def main(cfg):
     # env init
+    #ms.set_context(mode=cfg.system.mode, device_id=7)
```

Comment:
better to remove commented debugging code

Reply:
fixed. thanks.